### PR TITLE
Create slovak translation

### PR DESCRIPTION
**Pull request by petersoniq**
Add slovak localization.

### CODE REVIEW

File: src/translations/sk.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    "forest_fire": "lesný požiar",
+    "avalanches": "lavíny",
+    "rain": "dážď",
+    "flood": "povodeň",
+    "rain_flood": "dažďová povodeň"
+  },
+  "messages": {
+    "yellow": "Žlté varovanie - {0}",
+    "orange": "Ornažové varovanie - {0}",
+    "red": "Červené varovanie - {0}"
```
Comment (MrBartusek) [inline src/translations/sk.json:39]: JSON is not closed by `}`

**Review by MrBartusek — CHANGES_REQUESTED** [review]
Hey, thanks for contributing, however
- `sk.json` does not have a closing curly
- There is no mention of a new translation in README
- File is not imported in `src/localize.js`
- File is not added to `.github\workflows\translation.yaml`

Refer to: [CONTRIBUTING.md](https://github.com/MrBartusek/MeteoalarmCard/blob/master/CONTRIBUTING.md)

File: src/translations/sk.json (hunk 1 above)
Comment (petersoniq) [reply src/translations/sk.json:39]: Sorry for my mistake.
In the end missing:
   }
 }

**Comment by MrBartusek** [comment]
@petersoniq don't forget about other things from the review!

**Comment by petersoniq** [comment]
All done. I hope so.

**Comment by MrBartusek** [comment]
@petersoniq Hey, please make all the changes in the single PR, it will make the process much easier. You can push more commits to the branch that you made this PR from and they will appear here.

**Comment by MrBartusek** [comment]
There hasn't been any activity on this pull request recently so I will close it. Feel free to create a new one if you want to continue your work. Thanks for your contribution!

**Pull request closed without merging**